model/src/application/component/mod.rs

```diff
@@ -564,6 +588,12 @@ mod tests {
         button::ButtonStyle, select_menu::SelectMenuOption, text_input::TextInputStyle,
     };
     use serde_test::Token;
+    use static_assertions::assert_impl_all;
+
+    assert_impl_all!(Component: From<ActionRow>);
+    assert_impl_all!(Component: From<Button>);
+    assert_impl_all!(Component: From<SelectMenu>);
+    assert_impl_all!(Component: From<TextInput>);
```

Comment:
Let's simplify these into one assertion, then lgtm:
```suggestion
    assert_impl_all!(
        Component: From<ActionRow>,
        From<Button>,
        From<SelectMenu>,
        From<TextInput>
    );
```